File: estudo/neri/algoritmo/c/Triangulo.c
#include <stdio.h>
int main(){
    float li, ld, le;
    printf("Digite o comprimento: ");
    scanf("%f",&li);
    printf("Digite o comprimento: ");
    scanf("%f",&ld);
    printf("Digite o comprimento: ");
    scanf("%f",&le);
    if ((li < (le + ld)) && (le < (ld + li)) && (ld < (li + le)))
            if ((le == ld) && (le == li))
                printf("Triangulo equilatero");
            else if ((li == ld) || (li == le) || (le = ld))
                printf("Triangulo isosceles");
            else
                printf("Triangulo escaleno");
    else
        printf("Nao e um triangulo");
}

File: estudo/neri/algoritmo/java/Triangulo.java
import javax.swing.JOptionPane;

public class Triangulo {
    public static void main(String args[]){
        float li, ld, le;
        li = Float.parseFloat(JOptionPane.showInputDialog("Digite o comprimento: "));
        ld = Float.parseFloat(JOptionPane.showInputDialog("Digite o comprimento: "));
        le = Float.parseFloat(JOptionPane.showInputDialog("Digite o comprimento: "));
        if ((li < (le + ld)) && (le < (ld + li)) && (ld < (li + le)))
            if ((le == ld) && (le == li))
                    JOptionPane.showMessageDialog(null,"Triangulo equilatero");
            else if ((li == ld) || (li == le) || (le == ld))
                    JOptionPane.showMessageDialog(null,"Triangulo isosceles");
            else
                    JOptionPane.showMessageDialog(null,"Triangulo escaleno");
        else
                JOptionPane.showMessageDialog(null,"Nao e um triangulo");
    }
}
